### ISSUE TITLE
What does "affiliated" mean?

### ISSUE THREAD

**Issue by mgiuca** [WICG/WebApiDevice#6]
The explainer uses the term "affiliated" 5 times without further definition or elaboration. Please define it.

> If the current user is not affiliated, the promise is rejected

I assume (hope?) this means if the user is not enterprise enrolled, or the _domain administrator_ has not specifically allowed _this origin_ to use the API, the promise is rejected.

In other words, "the user is affiliated with _origin_" means the user is enterprise enrolled _and_ the domain administrator has specifically allowed _origin_ to use this API. You should explain that there would be a browser-specific process for an administrator to enable this API on certain origins.

**Comment by Ananubis** [comment]
Thank you, @mgiuca. Your understanding is the same as what we want to proposing.

I replaced the relevant 'affiliation' description with the 'trusted application' concept, which is introduced at the beginning of the explainer doc. Please help to check again if the current version looks better to you.

**Comment by mgiuca** [comment]
Thanks. I think that reads much more clearly now and is internally consistent.

However, the term "trusted application", especially "highly-trusted by the user agent" may overlap with the "high trust mode" concept that others within Chrome are working on. I don't think we want "highly trusted" to imply you get access to all of these APIs -- for example, we may wish to introduce a concept of "highly trusted" that is _not_ under enterprise control. However, the inverse should be true (you should need to be highly trusted to access these APIs).

Basically, "high trust" should be a prerequisite for access to the Managed Device API, not synonymous. So perhaps you should rename "trusted application" with "managed application", and somewhere explain that you have to be highly trusted to be a "managed application".

@reillygrant @slightlyoff

**Comment by mgiuca** [comment]
Quick ping for @reillyeon (I misspelled your username and had to edit it above; not sure if that will trigger mail).

**Comment by reillyeon** [comment]
+1 to removing "trusted application" from this explainer to avoid confusion with other efforts. I don't think it is necessary as we're specifically targeting managed devices and managed applications. There's no need to be circumspect about where the trust is coming from.

**Comment by reillyeon** [comment]
This was already raised in #1 and I'm not sure how to interpret @slightlyoff's comment here.

**Comment by sbykov-g** [comment]
Colleagues, I've addressed this issue in https://github.com/WICG/WebApiDevice/pull/22
